src/Facade/Services/ChangeRequestFacadeService.cs

```diff
@@ -195,6 +196,59 @@ public IResult<IEnumerable<ChangeRequestResource>> GetChangeRequestsRelevantToBo
                 changeRequests.Select(x => (ChangeRequestResource)this.resourceBuilder.Build(x, privileges)));
         }
 
+        public IResult<IEnumerable<ChangeRequestResource>> SearchChangeRequests(
+            string searchTerm,
+            bool? outstanding, 
+            int? lastMonths,
+            IEnumerable<string> privileges = null)
+        {
+            var changeRequests = new List<ChangeRequest>();
+
+            var fromDate = (lastMonths == null)
```

Comment:
could move into relevant part of IF statement. Also the hard coded date will date quickly. probably better to be relative to today